frontend/src/metabase/dashboard/context/context.tsx

```diff
@@ -194,7 +195,9 @@ const DashboardContextProviderInner = forwardRef(
       setTheme,
     } = useDashboardTheme(initTheme);
 
-    const shouldRenderAsNightMode = Boolean(isNightMode && isFullscreen);
+    const shouldRenderAsNightMode = Boolean(
+      isNightMode && (isFullscreen || isEmbeddingIframe),
+    );
```

Comment:
We should also render the pivot table cell background in night mode for static embedding. Not just for full-screen dashboards.